### DISCUSSION TITLE
<shift-e> / capital E not working in input fields

### DISCUSSION THREAD

**Discussion by Tillerz** [vladmandic/sdnext#1505] [General]
Hi,

see subject. I can't enter a capital E into any of the input fields in the UI.

**Comment by SoraEnkai** [comment]
Search for keybinds in settings, there's one for Shift + E, which is what you're experiencing here.
Edit: It's Agent Scheduler shortcut, which defaults to Shift + E
![image](https://github.com/vladmandic/automatic/assets/39128897/ce61eb84-f713-44d8-b6b2-c2f88e114004)

**Comment by brknsoul** [comment]
The author of Agent Scheduler is aware of this and new installs of the extension now default to Ctrl+E.

**Comment by andupotorac** [comment]
Still doesn't work

**Comment by gsgoldma** [comment]
use caps lock

**Reply by andupotorac** [reply]
Sure but that doesn't fix the bug.

**Comment by vladmandic** [comment]
is there upstream issue that i can track with agent scheduler?
